docs/04.INTERNALS.md

```diff
@@ -323,7 +324,7 @@ Collections are array-like data structures, which are optimized to save memory.
 
 ### Exception Handling
 
-In order to implement a sense of exception handling, the return values of JerryScript functions are able to indicate their faulty or "exceptional" operation. The return values are actually ECMA values (see section [Data Representation](#data-representation)) in which the error bit is set if an erroneous operation is occurred.
+In order to implement a sense of exception handling, the return values of JerryScript functions are able to indicate their faulty or "exceptional" operation. The return values are ECMA values (see section [Data Representation](#data-representation)) and if an erroneous operation is occurred an ECMA_VALUE_ERROR simple value is returned.
```

Comment:
AFAIK this can be simplified: "if an erroneous operation occurred the ECMA_VALUE_ERROR simple value is returned."

Reply:
Thank, I've applied you suggestion.